client-sdk/go/config/paratime.go

```diff
@@ -153,14 +164,17 @@ func (p *ParaTime) Namespace() common.Namespace {
 //
 // In case the given denomination does not exist, it provides sane defaults.
 func (p *ParaTime) GetDenominationInfo(d string) *DenominationInfo {
-	var di *DenominationInfo
-	if len(d) == 0 {
-		di = p.Denominations[NativeDenominationKey]
-	} else if di = p.Denominations[d]; di == nil {
-		di = p.Denominations[strings.ToLower(d)]
+	var (
+		di *DenominationInfo
+		ok bool
+	)
+	if d == "" {
```

Comment:
This method looks a bit dangerous, as if the ParaTime does not support consensus layer transfers, the call `p.GetDenominationInfo(p.ConsensusDenomination)` returns the default value without warning the caller.